### ISSUE TITLE
Weird Behaviour of 1.1.2.6 on the IOTASupport server

### ISSUE THREAD

**Issue by moodledidoodledi** [iotaledger/iri#73]
Hi, today I witnessed the following text running through the console log vey quickly:
<img width="1440" alt="bildschirmfoto 2017-02-21 um 13 32 55" src="https://cloud.githubusercontent.com/assets/23346705/23165320/d498a2ce-f83b-11e6-8997-5f50088aa5f2.png">
Node was not working anymore. A restart solved this. Any ideas?

**Comment by dhughesbc** [comment]
To reproduce: 

1. Issue this request: 
```
curl -d '{"command":"findTransactions", "addresses":["999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999"]}' http://localhost:14265
```

2. Kill the request immediately after issuing it, e.g. `^C`

Step #1 causes `final String response` to be about 20MB (Looks like a list of every transacton hash in the tangle?):
https://github.com/iotaledger/iri/blob/master/src/main/java/com/iota/iri/service/API.java#L510. 

Step #2 (client closing the http connection) causes the following block of code to throw a `ClosedChannelException` for every chunk that is written by `sinkChannel.write(responseBuf);`:

https://github.com/iotaledger/iri/blob/master/src/main/java/com/iota/iri/service/API.java#L525-L539 

If a chunk is 1 byte, that is ~20M exceptions based on current tangle size. 

In my case I originally found this issue by submitting an empty seed value using the iota JS libray, which apparently converts the seed value to 90 "9"s. I start my node like `java -jar iri-1.1.2.4.jar .... > iota.log 2>&1 &` and iota.log grew at about 100MB/min until I restarted the node.

**Comment by alfredch** [comment]
Solved in version 1.1.3.10
